### ISSUE TITLE
System unusable after extending "Case Mailbox Process Settings"

### ISSUE THREAD

**Issue by ianfp** [orocrm/crm#183]
Steps to reproduce:

1. Go to System -> Entities -> Entity Management
2. Edit "Case Mailbox Process Settings"
3. Check all four activities (Emails, Calendar Events, Calls, Tasks)
4. Save the changes
5. Press the red button to update the schema

On our production instance, the update failed with this error:
https://app.getsentry.com/share/issue/37343036302e313336343735363830/

Attempting to log in now results in this error:
https://app.getsentry.com/share/issue/37343036302e313336343739343830/

Running `oro:entity-extend:update-schema` and `cache:clear` commands does not solve the problem; indeed, `oro:entity-extend:update-schema --dry-run` doesn't even show any outstanding changes.

Interestingly, I tried to reproduce this on my laptop using a completely clean install, and the bug did not occur. So there's something about our production environment that is causing it.

Please let me know if I can provide any further info.

**Comment by artml** [comment]
@ianfp Thanks for highlighting this issue, Ian. I will check with our QA and they will get back to you with additional questions so we can investigate.

Just out of curiosity: What were you trying to achieve by enabling activities for case mailbox process settings entity? That one doesn't even have a view, let alone any meaningful activities. (They should be disabled by the way, and _that's_ definitely a bug.)

**Comment by ianfp** [comment]
@artml Thanks for getting back to me. It was my boss who originally encountered this problem; he says he was trying to "create a lead or opportunity from a case." I hope that's helpful; let me know what else I can do.

**Comment by ianfp** [comment]
BTW, I was able to restore our system from a database backup, so we're up and running with no problems. I'm bringing this to your attention in the hopes that we can contribute back to the project.

**Comment by artml** [comment]
@ianfp yes, bringing issues to our attention definitely helps us to fix them. Please don't stop the practice :)

Re your boss's request: to create leads or opportunities from a case you'd need to write a custom action; enabling activities won't help to achieve this. We would like to provide this functionality out of the box for our contact requests but this feature is still in the backlog and there's no clear implementation timeline yet.
